src/store/import-export/importExport.reducers.ts

```diff
@@ -0,0 +1,177 @@
+import { PayloadAction, createSelector, createSlice } from '@reduxjs/toolkit';
+
+import { Status } from '@/src/types/files';
+import {
+  LatestExportFormat,
+  Operation,
+  SupportedExportFormats,
+} from '@/src/types/importExport';
+
+import { RootState } from '..';
+
+interface UploadedAttachment {
```

Comment:
Is it possible to reuse existing classes for files?

Reply:
Thank you for pointing at this.
Existing interfaces reused.